frontend/src/NumberFormat.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import NumberFormat from 'react-number-format';
+import PropTypes from 'prop-types';
+
+export function NumberFormatCustom(props) {
+  const { inputRef, onChange, ...other } = props;
+
+  return (
+    <NumberFormat
```

Comment:
I'm not familiar with the react-number-format library. Is this doing something outside of adding a separator at the thousand marks?

Reply:
It makes it so that you cannot enter anything except for a number (eg if you try to type an "a", it won't show up in the input box)

Reply:
Can't you add `type="number"` to your `TextField` to require a number?

Reply:
Yep, that worked and is definitely simpler. For some reason I thought that attribute couldn't be applied in this case, but it can. I'll change that in the overall input form branch. Thank you!